content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -154,6 +151,15 @@ To deploy the Operator, follow the instructions available [here](../../#installa
    | **cert-manager** | This section configures the enablement of cert-manager. | - | - |
    | enabled | Enable/Disable deployment of cert-manager. Set to false if you already have cert-manager installed. | No | true |
 
+**Optional:**
+To enable reporting of trace data with [Zipkin](https://zipkin.io/), this can be achieved through the `csm-config-params` configMap in the sample CR or dynamically by editing the configMap.
```

Comment:
Replace "this can  be achieved through the" with "use"

Reply:
fixed.